sdcm/sla/sla_tests.py

```diff
@@ -105,16 +107,21 @@ def attach_sl_and_validate_scheduler_runtime(self, tester, new_service_level, ro
                                                  # restart_scylla parameter is temporary - wWorkaround for issue
                                                  # https://github.com/scylladb/scylla-enterprise/issues/2572
                                                  restart_scylla=False):
+        @retrying(n=15, sleep_time=1, message="Wait for service level has been attached to the role",
```

Comment:
Are we back to 15 seconds?

Reply:
No. it's new code, wait for SL is attache to the role, not propagated to the nodes

Reply:
I see it's new code.
I see that there are 15 retries of 1s, so the timeout for this wait remain 15s?

Reply:
We will wait for SL is attached to the role 15 sec, yes